.travis.yml

```diff
@@ -16,13 +16,11 @@ matrix:
     - php: 5.3
       env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.5
-      env: SYMFONY_VERSION=2.3.*
+      env: SYMFONY_VERSION=2.3.* SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.5
-      env: SYMFONY_VERSION=2.6.*
+      env: SYMFONY_VERSION=2.8.*
     - php: 5.5
-      env: SYMFONY_VERSION=2.7.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
+      env: SYMFONY_VERSION=3.0.*
```

Comment:
this should be added in the failure allowed matrix as well (there are 50 deprecation notices in 2.8, so there is no way this can work in 3.0):
```
matrix:
  include:
    # ...
    - php: 5.6
      env: SYMFONY_VERSION=3.0.*
  allowed_failure:
    - env: SYMFONY_VERSION=3.0.*
  fast_finish: true
```

Also,let's use PHP 5.6 for all matrices